modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase

```diff
@@ -107,4 +105,82 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-180358. Verify all status changes after 'need more info' request"
```

Comment:
@juliosilveira77 I`m trying a new approach for test description:
@description = "LPS-180358 - Organization status should changes after 'need more info' request' flow"

can you validate with gabi?